org.caveman250.gtkeddit.json

```diff
@@ -0,0 +1,163 @@
+{
+  "app-id": "org.caveman250.gtkeddit",
+  "version": "0.1",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "40",
+  "sdk": "org.gnome.Sdk",
+  "command": "gtkeddit",
+  "finish-args": ["--share=network", "--socket=wayland", "--socket=fallback-x11", "--socket=session-bus", "--filesystem=home" ],
+  "build-options" : {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": ["/include", "/lib/*/include", "/lib/pkgconfig", "/share/pkgconfig", "/share/aclocal", "/man", "/share/man", "/share/info", "/share/gtk-doc", "*.la", "*.a" ],
+  "modules": [
+    {
+      "name": "mm-common",
+      "cleanup": [ "/" ],
```

Comment:
Hm, does it actually work? I usually used `"*"` to remove all files from a module after build.

Reply:
yeah oops, fixed.